ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -13,42 +13,26 @@
 DEFAULT_SERVICE = 'elasticsearch'
 SPAN_TYPE = 'elasticsearch'
 
-# Original Elasticsearch class
-_Elasticsearch = elasticsearch.Elasticsearch
-
 
 def patch():
-    setattr(elasticsearch, 'Elasticsearch', TracedElasticsearch)
-
-def unpatch():
-    setattr(elasticsearch, 'Elasticsearch', _Elasticsearch)
-
 
-class TracedElasticsearch(wrapt.ObjectProxy):
-    """Traced Elasticsearch object
+    if getattr(elasticsearch, '_datadog_patch', False):
+        return
+    setattr(elasticsearch, '_datadog_patch', True)
+    wrapt.wrap_function_wrapper('elasticsearch.transport', 'Transport.perform_request', _perform_request)
```

Comment:
This one works fine until we use the default `Transport` that ships with ES. Indeed, you can check [it here][1] that the default client is expecting a `transport_class` that defaults to the `Transport` you're patching. I don't think we have to do anything here, but maybe you can just add a comment somewhere in this module (`patch.py`) just saying: "we're patching the default `elasticsearch.transport` module."

In that way if someone is using it's own `Transport` (for any reasons), they're aware to call this patch in their transport too.

[1]: https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/client/__init__.py#L156